src/components/StyledText/StyledLink/styled.ts

```diff
@@ -0,0 +1,16 @@
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+
+type Props = {
+	changeColor?: string
+}
+
+export const StyledLink = styled(Link)<Props>`
+	text-align: center;
+	text-decoration: none;
+	text-transform: uppercase;
+	color: ${({ changeColor }) =>
+		changeColor === 'darkBlue'
+			? ({ theme }) => theme.text.secondary // Yellow color
+			: ({ theme }) => theme.text.link}; // Default Link color
```

Comment:
This comment I mean `Yellow color` seem unnecessary to me.

- These properties is dynamic.
- If we add new thema or changed old, we must edit this comments too. 
- Comments should not hamper the sustainability of the code